bait-files/src/main/java/com/google/sps/servlets/BlobstoreServeImageServlet.java

```diff
@@ -0,0 +1,28 @@
+package com.googl.sps.servlets;
+
+import com.google.appengine.api.blobstore.BlobKey;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Serves the image file corresponding to the given blob key
+ */
+@WebServlet("/blobstore-get-image")
```

Comment:
The name of this should probably match the name of the servlet.

Reply:
Changed the servlet name